corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -419,16 +419,17 @@ def indicators(self, columns, filters, is_multiselect_chart_report=False):
                 indicator = property_.to_report_filter_indicator(filter_)
                 indicators.setdefault(str(indicator), indicator)
 
+        if as_dict:
+            return indicators
+
         return list(indicators.values())
```

Comment:
nit: might be slightly more readable if you had:
`return indicators if as_dict else list(indicators.values())`